server/server.go

```diff
@@ -0,0 +1,128 @@
+// main package to be able to run the StudentsServer for now
+package main
+
+import (
+	"context"
+	"net"
+
+	spb "github.com/BetterGR/students-microservice/students_protobuf"
+	"google.golang.org/grpc"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	// define port
+	address            = "localhost:50052"
+	connectionProtocol = "tcp"
+)
+
+// StudentsServer is an implementation of GRPC Students microservice.
+type StudentsServer struct {
+	// throws unimplemented error
+	spb.UnimplementedStudentsServiceServer
+}
+
+// GetStudent search for the Student that corresponds to the given id and returns them.
+func (s *StudentsServer) GetStudent(ctx context.Context,
+	req *spb.GetStudentRequest) (*spb.GetStudentResponse, error) {
+	logger := klog.FromContext(ctx)
+	logger.Info("Received GetStudent request for:", "student_id", req.Id)
```

Comment:
`logger.Info` does not build a string in the sense of concatenation. The main message should describe the main event and the following key-value tuples add more context.

Therefore it should be `logger.Info("Received GetStudent request", "studentID", req.Id)` - note the camelCase as well.